src/dev/src/app/timeline/timeline.demo.html

```diff
@@ -28,7 +28,7 @@ <h4>Horizontal timeline</h4>
   </li>
   <li class="clr-timeline-step gemini-ignore">
     <div class="clr-timeline-step-header">11:59 am</div>
-    <clr-icon shape="spinner" aria-label="In progress"></clr-icon>
+    <clr-spinner clrMedium aria-label="In progress">Fetching data</clr-spinner>
```

Comment:
So you're replacing the icon with an actual spinner? I approve.

Reply:
#dogfooding